### PR TITLE
[BUG] After trigger pausePushData, new added channel still can push data

### LINKED ISSUE #375: [BUG] After trigger pausePushData,  new added channel still can push data
### What is the bug?
When MemoryTracker trigger pausePushData
截屏2022-08-17 下午3 14 37
ChannelLimiter call method `onPause`
```
  @Override
  public void onPause(String moduleName) {
    if (this.moduleName.equals(moduleName)) {
      logger.info(this.moduleName + " channels pause read.");
      pauseAllChannels();
    }
  }
```

but don't paused new added 
```
  @Override
  public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
    channels.add(ctx.channel());
    super.handlerAdded(ctx);
  }
```

So worker still receive PushData/PushMergedData and push server memory increased and OOM
```
22/08/17 07:41:36,633 INFO [MemoryTracker-action-thread] MemoryTracker: Trigger pausePushDataAndResumeReplicate action
22/08/17 07:41:36,634 INFO [MemoryTracker-action-thread] ChannelsLimiter: push channels pause read.
22/08/17 07:41:36,765 INFO [MemoryTracker-action-thread] ChannelsLimiter: replicate channels resume read.
22/08/17 07:41:36,766 INFO [MemoryTracker-action-thread] MemoryTracker: Trigger trim action
22/08/17 07:41:36,774 INFO [MemoryTracker-action-thread] MemoryTracker: Trigger trim action
22/08/17 07:41:36,781 INFO [MemoryTracker-action-thread] MemoryTracker: Trigger trim action
```
截屏2022-08-17 下午3 25 27

### How to reproduce the bug?
Steps to reproduce the bug.

### Could you share logs or screenshots?
If applicable, add logs/screenshots to help explain your problem.

/cc @who-need-to-know

/assign @who-can-solve-this-bug

### PR DESCRIPTION
**Pull request by FMX**
# [BUG] After trigger pausePushData, new added channel still can push data

### Related issues.
closes #375 

### How was this patch tested?

/cc @waitinfuture @AngersZhuuuu

### CODE REVIEW

File: common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java [hunk 1]
```diff
@@ -65,6 +69,9 @@ private void resumeAllChannels() {
   @Override
   public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
     channels.add(ctx.channel());
+    if (isPaused.get()) {
+      ctx.channel().config().setAutoRead(false);
```
Comment (AngersZhuuuu) [inline common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java:73]: If `resumeAllChannels` running between `if (isPaused.get()) {` and this line, some channel will never be set to true?

should we add `sync`?

Comment (FMX) [reply common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java]: Yes, there is a possibility.

**Review by AngersZhuuuu — APPROVED** [review]
Trigger pause and resume is not frequent, LGTM

File: common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java (hunk 1 above)
Comment (FMX) [reply common/src/main/java/com/aliyun/emr/rss/common/network/server/ChannelsLimiter.java:73]: I re-checked the logic. The possibility is zero. If `resumeAllChannels` running between `if (isPaused.get()) {` ,  resumeAllChannels function will correct the channel's auto read configuration.

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**